tedana/workflows/tedana.py

```diff
@@ -331,6 +344,10 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
     if not op.isdir(out_dir):
         os.mkdir(out_dir)
 
+    io.outdir = out_dir
+    io.set_prefix(prefix)
+    io.set_convention(convention)
```

Comment:
You have a problem that bothered me earlier. You are running this functions in `io` and assigning the new value to `convention` and `prefix` in those functions Those new values do not seem to leave those functions. When I suggested a change here, I had: `io.convention = io.set_convention(convention)` and that function ended with `return convention` I don't know if there is a better way to do it, but that worked and what you have doesn't. If you run the workflow with `--convention orig` it still runs with the default bids.

Reply:
Try [inserting the global keyword](https://github.com/notZaki/tedana/commit/4ec51d717c4bb34dc99729e935cc7b8755d7f9e3) in `set_function`.